hack/build-manifests.sh

```diff
@@ -351,7 +351,7 @@ EOM
 
 IMAGE_NAME=$(get_image_digest "${OPERATOR_IMAGE}")
 conversionContainer=$(get_image_digest "${CONVERSION_CONTAINER}")
-vmwareContainer=$(get_image_digest "${VMWARE_CONTAINER}")
+vmwareContainer=$(get_image_digest "${IMPORT_PROVIDER_CONTAINER}")
```

Comment:
do we want to change this variable as well? and ims-vmware-image-name flag?

Reply:
updated